src/templates/list-table-block/action-column.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div
     class="max-w-full"
+    :class="isTabs ? 'mt-4' : ''"
```

Comment:
Move pra uma computed e retorna as classes necessárias

```suggestion
    class="max-w-full"
    :class="isTabs ? 'mt-4' : ''"
```